src/app/cli/src/tests/coda_peers_test.ml

```diff
@@ -5,7 +5,7 @@ let name = "coda-peers-test"
 
 let main () =
   let%bind program_dir = Unix.getcwd () in
-  let n = 3 in
+  let n = 1 in
```

Comment:
why did this change?

Reply:
oh whoops, that accidentally got in. Thanks